generate-commitish.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#Generates a commitish string for climate explorer and stores it in 
+# the CE_CURRENT_VERSION environment variable.
+
+VERSIONTAG="$(git describe --tags --abbrev=0)"
```

Comment:
If we intend to use git, we'll have to ensure that git is included in whatever Docker base we have.

Turns out that it is (but that actually surprises me):

```
hiebert@docker-dev01:~$ docker run -it node:latest git --version
git version 2.1.4
```